modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminManagementToolbarDisplayContext.java

```diff
@@ -122,15 +122,10 @@ public String getClearResultsURL() {
 				"status", WorkflowConstants.STATUS_ANY
 			);
 
-		List<ContentDashboardItemFilterProvider>
-			contentDashboardItemFilterProviders =
-				_contentDashboardItemFilterProviderRegistry.
-					getContentDashboardItemFilterProviders();
-
 		try {
 			for (ContentDashboardItemFilterProvider
 					contentDashboardItemFilterProvider :
-						contentDashboardItemFilterProviders) {
+						_contentDashboardItemFilterProviders.toList()) {
```

Comment:
Hey @Valmir-unicap, `ServiceTrackerList` is an iterable. So, I think you don't need to call the `.toList` here. Please check that.

Thx